astroplan/tests/test_scheduling.py

```diff
@@ -75,7 +75,7 @@ def test_schedule_insert_slot():
     end_time = start + duration
     block = TransitionBlock.from_duration(duration)
     schedule.insert_slot(end_time - duration, block)
-    assert not end_time - duration == start
+    assert end_time - duration == start
```

Comment:
@bmorris3 @wtgee -- I wasn't sure about this change but had to make it to get tests passing. Was it still the expectation that `end_time - duration != start`?

Reply:
Yeah, I'm confused as to why that was passing before. @bmorris3?

Reply:
I'm also stumped by this. The change was introduced in https://github.com/astropy/astroplan/commit/221d0d68bc23f82d540e22ffa8e8e5b5137655ab.